.gitignore

```diff
@@ -58,5 +58,6 @@ __build__/**
 *.swp
 .vscode
 
-# Proxy config #
+# Configurations #
 proxy-conf.js
+/src/config/config.json
```

Comment:
maybe ignore the whole directory?

Reply:
The directory contains the timezones.json file, so we don't need to ignore whole directory